docs/chaos-engineering/chaos-faults/aws/ebs-loss-by-tag.md

```diff
@@ -4,8 +4,8 @@ title: EBS Loss By Tag
 ---
 
 ## Introduction
-- It causes chaos to disrupt state of ebs volume by detaching it from the node/ec2 instance for a certain chaos duration using volume tags.
-- In case of EBS persistent volumes, the volumes can get self-attached and experiment skips the re-attachment step.
+- It causes chaos to disrupt state of ebs volume by detaching it from the node/EC2 instance for a certain chaos duration using volume tags.
+- In case of EBS persistent volumes, the volumes can get self-attached and  fault skips the re-attachment step.
```

Comment:
```suggestion
- In case of EBS persistent volumes, the volumes can get self-attached and the re-attachment step is skipped.
```
At other places too